packages/plugin-ext/src/plugin/authentication-ext.ts

```diff
@@ -163,30 +119,8 @@ export class AuthenticationExtImpl implements AuthenticationExt {
         throw new Error(`Unable to find authentication provider with handle: ${providerId}`);
     }
 
-    $onDidChangeAuthenticationSessions(id: string, label: string): Promise<void> {
-        this.onDidChangeSessionsEmitter.fire({ provider: { id, label } });
+    async $onDidChangeAuthenticationSessions(provider: theia.AuthenticationProviderInformation): Promise<void> {
+        this.onDidChangeSessionsEmitter.fire({ provider });
         return Promise.resolve();
```

Comment:
Can be deleted if the function is declared `async`.

Reply:
True, thank you!